packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -1,45 +1,84 @@
+import useSidebarRendered from '../../hooks/useSidebarRendered';
+import SharePostBar from './SharePostBar';
 
 type SquadPageHeaderProps = {
   squad: Squad;
   members: SquadMember[];
   onNewSquadPost: () => void;
 };
 
+const MAX_WIDTH = 'laptop:max-w-[38.5rem]';
+
 export function SquadPageHeader({
   squad,
   members,
   onNewSquadPost,
 }: SquadPageHeaderProps): ReactElement {
+  const { sidebarRendered } = useSidebarRendered();
+
   return (
-    <section className="flex flex-col items-center px-6 pb-0 tablet:pb-10 mb-6 w-full tablet:border-b min-h-20 border-theme-divider-tertiary">
-      <SquadImage
-        className="mt-4 w-16 tablet:w-24 h-16 tablet:h-24"
-        {...squad}
-      />
-      <h3 className="mt-4 tablet:mt-6 font-bold typo-title2">{squad.name}</h3>
-      <h4 className="mt-1 tablet:mt-2 typo-body text-theme-label-tertiary">
-        @{squad.handle}
-      </h4>
-      <p className="mt-5 tablet:mt-4 w-full text-center typo-body text-theme-label-secondary max-w-[42rem]">
+    <FlexCol className="relative items-center laptop:items-start px-6 pb-20 tablet:pb-20 laptop:pb-14 mb-6 w-full tablet:border-b laptop:px-[4.5rem] min-h-20 border-theme-divider-tertiary">
+      <div className="flex flex-col laptop:flex-row items-center">
+        <SquadImage className="w-16 tablet:w-24 h-16 tablet:h-24" {...squad} />
+        <FlexCol className="mt-4 laptop:mt-0 ml-6">
+          <h3 className="font-bold typo-title2">{squad.name}</h3>
+          <h4 className="mt-1 tablet:mt-2 typo-body text-theme-label-tertiary">
+            @{squad.handle}
+          </h4>
+        </FlexCol>
+      </div>
+      <p
+        className={classNames(
+          'mt-6 w-full typo-body text-theme-label-tertiary',
+          MAX_WIDTH,
+        )}
+      >
         {squad.description}
       </p>
+      {!sidebarRendered && (
+        <SquadMemberShortList
+          squad={squad}
+          members={members}
+          memberCount={squad.membersCount}
+          className="my-6"
+        />
+      )}
       <SquadHeaderBar
-        className="mt-6"
         squad={squad}
         members={members}
         memberCount={squad.membersCount}
         onNewSquadPost={onNewSquadPost}
+        className={sidebarRendered && 'absolute top-0 right-[4.5rem]'}
       />
       <EnableNotification
         contentName={squad.name}
         source={NotificationPromptSource.SquadPage}
+        className={classNames('w-full', MAX_WIDTH)}
       />
-    </section>
+      <div
+        className={classNames(
+          'absolute bottom-0 w-full translate-y-1/2 px-6 laptop:px-0 ',
+          MAX_WIDTH,
+        )}
+      >
+        <SharePostBar
+          className={classNames(
+            'w-full z-1 relative',
+            'before:bg-theme-bg-primary before:w-full before:h-full before:absolute before:inset-0 before:-z-2',
+            'after:bg-theme-float after:w-full after:h-full after:absolute after:inset-0 after:-z-1',
```

Comment:
Our `bg-theme-float` has a built-in opacity property, which leads the background to be visible, but the requirement is to make it **not** visible like below:
![image](https://user-images.githubusercontent.com/13744167/226790410-88f3e463-4e28-49cb-a415-ae27ffc8032e.png)

Without the before and after by using just `bg-theme-float`, it looks like this:
![image](https://user-images.githubusercontent.com/13744167/226790747-f135d40c-4f02-4d02-be45-3d224c6a6a6b.png)

Reply:
Found a cleaner workaround! 🚢